scripts/lua/modules/ts_5min_dump_utils.lua

```diff
@@ -187,7 +187,48 @@ function ts_dump.host_update_stats_rrds(when, hostname, host, ifstats, verbose)
   ts_utils.append("host:unreachable_flows", {ifid = ifstats.id, host = hostname,
 					   flows_as_client = host["unreachable_flows.as_client"],
 					   flows_as_server = host["unreachable_flows.as_server"]},
-		  when, verbose)
+      when, verbose)
+  
+  interface.select(tostring(ifstats.id))
```

Comment:
why the selection again?

Reply:
I think for a correct call to interface.getHostInfo(). in other scripts the are called together. isn't it necessary?

Reply:
interface is already selected

Reply:
ok, thank you